posthog/models/team/team.py

```diff
@@ -175,8 +175,6 @@ class Team(UUIDClassicModel):
         on_delete=models.CASCADE,
         related_name="teams",
         related_query_name="team",
-        null=True,
```

Comment:
Looks like there are fixtures in the plugin-server tests that need to be updated as they don't account for this not being nullable anymore